### PR TITLE
Update device support on website.

### PR DESCRIPTION
**Pull request by lah7**
When you're ready for release -- device support changes:

#### Supported:
+ Oranta
+ Razer Core

#### In Progress:
+ Naga Hex

### CODE REVIEW

**Comment by z3ntu** [comment]
The Razer Core image should be a bit smaller tbh
Also the Kraken v1 picture can be updated with that for example:

**Comment by lah7** [comment]
Thanks - yeah, an oversight about the large Razer Core image.

**Comment by z3ntu** [comment]
* Some links to the store pages are like `/gb-en/store/` where others are simply `/store/` #consistency

* We still have to find a nice pic for the mug^^ but Razer doesn't seem to have nice promotional pictures for it...

File: index.html [hunk 1]
```diff
@@ -194,7 +194,7 @@
                   <a href="http://www.razerzone.com/store/razer-deathstalker-chroma" target="_blank">
                     <img src="https://assets.razerzone.com/eeimages/products/22563/rzr_deathstalker_chroma_03__store_gallery.png">
                     <br>
-                    <h5>DeathStalker Chroma</h5>
+                    <h5>Deathstalker Chroma</h5>
```
Comment (z3ntu) [inline index.html]: "DeathStalker" was correct

Comment (lah7) [reply index.html]: You're right, thanks for the spot -- I was basing it on what Razer had in the title bar for a different DeathStalker:

image

 :confused: 

image

**Comment by z3ntu** [comment]
`sudo layman -a vifino-overlay` instead of `sudo layman -f -o https://github.com/vifino/vifino-overlay/raw/master/vifino-overlay.xml -a vifino-overlay` now 👍 

CC: @vifino

**Comment by vifino** [comment]
:tada: :tada: :tada:

**Comment by z3ntu** [comment]
Solus users should be able to install the driver by doing
```
sudo eopkg install razer-drivers
```
I am not sure who exactly packages that but I guess it's the Solus maintainers.

p.s. it seems that the user isn't added to the plugdev group automatically.

**Pull request merged**